.ci/jenkins/dsl/jobs.groovy

```diff
@@ -19,15 +20,52 @@ Map getMultijobPRConfig() {
                 primary: true,
                 // TODO remove once https://issues.redhat.com/browse/KOGITO-4113 is done 
                 // as it will become the default path
-                jenkinsfile: OPTAPLANNER_JENKINSFILE_PATH
+                jenkinsfile: BUILD_CHAIN_JENKINS_PATH,
```

Comment:
As Jenkinsfile/repo_url is centralize for Buildchain, I think we could just add a `use_buildchain` parameter (same level as `parallel: true`) in multijob config and update this [method](https://github.com/kiegroup/kogito-pipelines/blob/master/dsl/seed/src/main/groovy/org/kie/jenkins/jobdsl/templates/KogitoJobTemplate.groovy#L319)  to set correct PR configuration with jenkinsfile and repo url to checkout if `use_buildchain` is true
wdyt ?
That way we don't repeat the information for all repos to check

Reply:
done by https://github.com/kiegroup/optaplanner/pull/1440/commits/d03551ccae8b3bdfbc8140a551861beae5779f4b